### ISSUE TITLE
[reg/doc] regtool hasn't documented all features

### ISSUE THREAD

**Issue by sjgitty** [lowRISC/opentitan#6]
I noticed that `no_auto_intr_regs` is not documented at this time. There may be others, we should contrast the tool vs. the documentation and itemize here and fix.

**Comment by tjaychen** [comment]
i'm tagging all the people i can remember who have probably touched the regtool in some form... we should probably do this before M3 or something.

**Comment by msfschaffner** [comment]
yeah there are several undocumented features by now (although I think some things like the CM labels have been documented). we should do a pass before M3 as you suggest.

**Comment by rswarbrick** [comment]
I'm going to close this issue because `reggen` *does* have some documentation and it's not really clear to me that this issue serves much purpose.

The documentation has also improved a bit since this issue was created in 2019(!!!). See https://opentitan.org/book/util/reggen/index.html for the rendered current version.
